lib/msal-browser/src/cache/BrowserCacheManager.ts

```diff
@@ -400,26 +400,48 @@ export class BrowserCacheManager extends CacheManager {
      * Gets the active account
      */
     getActiveAccount(): AccountInfo | null {
-        const activeAccountIdKey = this.generateCacheKey(PersistentCacheKeys.ACTIVE_ACCOUNT);
-        const activeAccountId = this.browserStorage.getItem(activeAccountIdKey);
-        if (!activeAccountId) {
+        const activeAccountKeyHomeLocalIds = this.generateCacheKey(PersistentCacheKeys.ACTIVE_ACCOUNT_HOME_LOCAL_IDS);
+        const activeAccountValueHomeLocalIds = this.getItem(activeAccountKeyHomeLocalIds);
+        if (!activeAccountValueHomeLocalIds) { 
+            // if new active account cache type isn't found, it's an old version, so look for that instead
+            const activeAccountKeyLocal = this.generateCacheKey(PersistentCacheKeys.ACTIVE_ACCOUNT);
+            const activeAccountValueLocal = this.getItem(activeAccountKeyLocal);
+            if(!activeAccountValueLocal) {
+                return null;
+            }
+            const activeAccount = this.getAccountInfoByFilter({localAccountId: activeAccountValueLocal})[0] || null;
+            if(activeAccount) {
+                this.setActiveAccount(activeAccount); // update old active account cache type to new type
```

Comment:
Can we also remove the old cache schema here?

Reply:
Jason suggested not to, in case we roll back

Reply:
Sounds good, fixed!